app/core/database.py

```
@@ -0,0 +1,70 @@
+            raise
+
+    async def connect_redis(self):
+        """Connect to Redis"""
+        try:
+            self.redis_client = redis.from_url(
+                settings.redis_url, decode_responses=True
+            )
+            # Test connection
+            self.redis_client.ping()
+            logger.info("Successfully connected to Redis!")
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)